_docs/csv-use-case.md

```diff
@@ -0,0 +1,420 @@
+        </sourceAttributes -->
+        <addAttributes>
+            <att name="colorBarMaximum" type="double">8000.0</att>
+            <att name="colorBarMinimum" type="double">-8000.0</att>
+            <att name="colorBarPalette">TopographyDepth</att>
+            <att name="ioos_category">Location</att>
+            <att name="long_name">Depth</att>
+            <att name="standard_name">depth</att>
+            <att name="units">m</att>
+        </addAttributes>
+    </dataVariable>
+    <dataVariable>
+        <sourceName>TEMPST01</sourceName>
+        <destinationName>TEMPST01</destinationName>
+        <dataType>double</dataType>
+        <!-- sourceAttributes>
+        </sourceAttributes -->
+        <addAttributes>
+            <att name="_FillValue" type="double">NaN</att>
+            <att name="ioos_category">Unknown</att>
+            <att name="long_name">TEMPST01</att>
+        </addAttributes>
+    </dataVariable>
+    <dataVariable>
+        <sourceName>PSALST01</sourceName>
+        <destinationName>PSALST01</destinationName>
+        <dataType>float</dataType>
+        <!-- sourceAttributes>
+        </sourceAttributes -->
+        <addAttributes>
+            <att name="ioos_category">Unknown</att>
+            <att name="long_name">PSALST01</att>
+        </addAttributes>
+    </dataVariable>
+    <dataVariable>
+        <sourceName>SSALST01</sourceName>
+        <destinationName>SSALST01</destinationName>
+        <dataType>double</dataType>
+        <!-- sourceAttributes>
+        </sourceAttributes -->
+        <addAttributes>
+            <att name="_FillValue" type="double">NaN</att>
+            <att name="ioos_category">Unknown</att>
+            <att name="long_name">SSALST01</att>
+        </addAttributes>
+    </dataVariable>
+</dataset>
+```
+Notice the `time` variable and it's associated configuration information:
+```xml
+    <dataVariable>
+        <sourceName>time</sourceName>
+        <destinationName>time</destinationName>
+        <dataType>String</dataType>
+        <!-- sourceAttributes>
+        </sourceAttributes -->
+        <addAttributes>
+            <att name="ioos_category">Time</att>
+            <att name="long_name">Time</att>
+            <att name="standard_name">time</att>
+            <att name="time_precision">1970-01-01T00:00:00Z</att>
+            <att name="units">yyyy-MM-dd&#39;T&#39;HH:mm:ss&#39;Z&#39;</att>
+        </addAttributes>
+    </dataVariable>
+```
+
+
+* Append xml to the end of datasets.xml
+  * See /python_tools/script1insert.py
+  * Master datasets file is at /usr/local/erddap-gold-standard/erddap/content/datasets.xml
+  * Be sure to put it above the closing </erddapDatasets> tag. Don’t worry, the script takes care of that.
+* Flag the dataset for ERDDAP to load
+  * Eg.$ touch erddap/data/flag/2019_asset_inventory
+* Check ERDDAP for the new dataset (might take a little if it’s a large dataset)
+
+```
+/usr/local/erddap-gold-standard/python_tools$ python
```

Comment:
my `~/erddap-gold-standard` repository cloned from ioos doesn't have `~/erddap-gold-standard/python_tools`...

Reply:
yeah, that's something special I built. I'll remove those pieces for now.